src/app/tests/suites/certification/Test_TC_MOD_1_1.yaml

```diff
@@ -35,35 +35,57 @@ tests:
       command: "readAttribute"
       attribute: "ClusterRevision"
       response:
-          value: 1
+          value: 2
```

Comment:
This might cause issues when running this YAML against downrev devices.  Need to figure out with the cert folks how this is supposed to work.

Reply:
@bzbarsky-apple based on an earlier call with the CSG Test Plan TT - the test scripts need to match the current cluster versions, so preserving the backwards compatibility (new script + old cluster rev) is not required. 

@raju-apple please let us know if you agree